product_docs/docs/biganimal/release/getting_started/preparing_cloud_account/01_preparing_azure/index.mdx

```diff
@@ -8,11 +8,12 @@ BigAnimal requires you to check the readiness of your Azure subscription before
 
 -   Are the necessary Azure resource providers registered for your subscription?
 -   Is there a restriction on SKUs for the standard Esv3 family and standard D2_v4 VM size?
--   Is there a sufficient limit on the number of vCPU or public IP addresses in your region?
+-   Is there a sufficient limit on the number of vCPU or public IP addresses in your region?    
 
+## Prerequisites
+When preparing your Azure account, make sure that you're assigned either the Global Administrator role or the Privileged Role Administrator role in Azure AD and that you have the Owner role for your BigAnimal Azure subscription.
```

Comment:
Is there something like "Privilege Role Administrator Role"?? can we have all letters in lower case??
```suggestion
When preparing your Azure account, make sure that you're assigned either the Global Administrator role or the Privileged Role Administrator role in Azure AD and that you have the Owner role for your BigAnimal Azure subscription.
```

Reply:
It looks like Microsoft uses upper case in their Azure documentation, and we also use this capitalization in [this BigAniimal](https://www.enterprisedb.com/docs/biganimal/latest/getting_started/02_connecting_to_your_cloud/connecting_azure/#prerequisites). Of course, I can make it all lower case if you think it's better.